index.js

```diff
@@ -10,9 +10,317 @@ function ask(questionText) {
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  let j = 0; //j is a switch variable. it changes based off of different responses so that the next block of code knows the implication of the previous response.
```

Comment:
j is a very opaque variable name. Consider using a more descriptive name such as switchVariable, or responseCheck to improve readability